docs/kubernetes/user-guides/pre-provisioned.md

```diff
@@ -0,0 +1,63 @@
+# Kubernetes Pre-Provisioned Disks User Guide
+This guide gives a simple example on how to use this driver with disks that have been pre-provisioned by an administrator.
+
+## Install Driver
+
+See [instructions](driver-install.md)
+
+## Pre-Provision a Disk
+
+If you have not already pre-provisioned a disk on GCP you can do that now.
+
+1. Create example PD
+```
+$ gcloud compute disks create test-disk --zone=us-central1-c --size=5Gi
+```
+
+## Create Persistent Volume for Disk
+
+1. Create example Zonal Storage Class
+```
+$ kubectl apply -f ./examples/kubernetes/demo-zonal-sc.yaml
+```
+
+2. Change PROJECT variable to your project and create example Persistent Volume
+```
+$ cat <<EOF | kubectl apply -f -
+apiVersion: v1
+kind: PersistentVolume
+metadata:
```

Comment:
There's actually one outstanding issue where you need to add the "provisioned-by" annotation to the PV in order for the CSI driver to handle deletion of the volume. https://github.com/kubernetes-csi/external-provisioner/issues/202